docs/current_docs/manuals/administrator/engine/272381-custom-registry.mdx

```diff
@@ -0,0 +1,42 @@
+---
+slug: /manuals/administrator/272381/custom-registry
+---
+
+# Configure a Custom Registry Mirror
+
+The Dagger Engine can be configured to use a different registry mirror for container images instead of the default (Docker Hub).
```

Comment:
```suggestion
The Dagger Engine can be configured to use container registry mirrors for any registry URL. This allows container images that refer to one registry to instead be redirected to a different one.
```

Just want to clarify that there isn't a default; this feature lets you setup a mirror for any registry url.